pkg/cyclonedx/cyclonedx_test.go

```diff
@@ -66,3 +66,31 @@ func TestToDraconSaaSInfra(t *testing.T) {
 	require.NoError(t, json.Unmarshal([]byte(*issues[0].CycloneDXSBOM), &sbom2))
 	assert.Equal(t, sbom1, sbom2)
 }
+
+func TestToDraconTargetOverride(t *testing.T) {
+	rawSaaSBOM, err := os.ReadFile("./testdata/saasBOM.json")
+	require.NoError(t, err)
+
+	issues, err := ToDracon(rawSaaSBOM, "json", "my-awesome-infra")
+	require.NoError(t, err)
+
+	saasBOM := string(rawSaaSBOM)
+	expectedIssues := []*v1.Issue{
+		{
+			Target:        "my-awesome-infra",
+			Type:          "SBOM",
+			Title:         "SBOM for my-awesome-infra",
+			Severity:      v1.Severity_SEVERITY_INFO,
+			CycloneDXSBOM: &saasBOM,
+		},
+	}
+	assert.Equal(t, expectedIssues[0].Target, issues[0].Target)
```

Comment:
we should use require instead of assert to immediately terminate the tests with a message about what failed